lib/customErrorHandler.js

```diff
@@ -3,14 +3,19 @@ const debug = require('debug')('swagger-endpoint-validator:custom-error-handler'
 /**
  * Adds a custom error handler for the errors thrown by the validator.
  * @param {Object} app - Express application object.
+ * @param {Object} options - Configuration options.
  * @see https://github.com/cdimascio/express-openapi-validator#asyncawait
  */
-const add = app => {
+const add = (app, options) => {
 	debug('Adding custom error handler...');
 
-	app.use((error, _req, res, _next) => { // eslint-disable-line no-unused-vars
+	app.use((error, _req, res, next) => { // eslint-disable-line no-unused-vars
 		debug(JSON.stringify(error));
-		res.status(error.status || 500).send(error);
+		if (options.propagateError) {
```

Comment:
Will we always have a non null options object?

Reply:
I'm passing in the `normalizedOptions`, which always returns an object according to `lib/normalizeOptions.js`